src/agl/WebAppManagerServiceAGL.h

```diff
@@ -17,6 +17,31 @@ constexpr char kSendAglReady[] = "ready-event";
 class WamSocket;
 class WamSocketLockFile;
 
+struct startup_args {
```

Comment:
Could you move startup_args to be inside WebAppManagerAGL? Then you should use CamelCase for naming. It would be WebAppManagerServiceAGL::StartupArgs